### ISSUE TITLE
StorageUpgrade Multiplier does not work correctly

### ISSUE THREAD

**Issue by SDXC** [jaquadro/StorageDrawers#124]
I have set different StorageUpgrade multipliers in the config (2,4,8,16,32) but they don't seem to work correctly. The Full Drawer stops at 11.6K items and I can't put more items in it even though the max count should be higher.

I'm using StorageDrawers-1.7.10-1.5.12.jar

**Comment by jaquadro** [comment]
Already-placed drawers hold their original storage values.  If you break and replace them, you should get the new capacities.

**Comment by SDXC** [comment]
Just for confirmation:
I set the multipliers in the config, started Minecraft, created a new world, crafted a full drawer and placed it. After that I put 1024 blocks of dirt in it, applied a storage upgrade (III) (multiplier: 4), put up to 4096 blocks, applied a second storage upgrade (III) and so on. Now I have a full drawer with three level III upgrades and a level IV (multiplier: 8) which should extend the maximum to 20480 blocks, but its stuck on 11.6K.

**Comment by jaquadro** [comment]
Okay, I miss-read that as modifying the base values, which is where the existing drawer problem has tripped up before.  No idea how the 11.6k is getting derived, so I need to look into it.

**Comment by SDXC** [comment]
Aaargh... I'm sooo sorryyy!
The problem was the dirt block itself. Somehow I ended up having 2 dirt block stacks with different origins but exactly the same texture and name. When I tried to put the foreign dirt stack in the drawer it didn't work because it's not registered in the ore dictionary. When I tried to reproduce the problem yesterday I always used the wrong dirt stack.

Again, I'm sorry. But thanks for looking into it. You're doing a great job and I really enjoy your mod.
